components/popover/stories/template.js

```diff
@@ -72,31 +71,37 @@ export const Template = ({
 				let yOffset = "+ 0px";
 				if (position.includes("top") || position.includes("bottom") && !(position.includes("-top") || position.includes("-bottom"))) {
 					x = triggerXCenter - (popWidth > 0 ? popWidth / 2 : popWidth);
-				} 
+				}
 				if (position.includes("left") || position.includes("right")) {
 					y = triggerYCenter - (popHeight > 0 ? popHeight / 2 : popHeight);
 				}
 				if (position.includes("top") && !position.includes("-top")) {
 					y = rect.top - popHeight;
-					yOffset = "- var(--spectrum-popover-offset)";
+					yOffset = withTip
+						? "- (var(--spectrum-popover-pointer-height) + var(--spectrum-popover-animation-distance))"
```

Comment:
When looking at the storybook variant with tip, it appears as if the tip to source button is measuring 9px instead of 8px. Is it possible we need to also be accounting for the border on the tip?

Without tip, measures 8px from source to border:
![Screenshot 2023-11-03 at 9 12 05 AM](https://github.com/adobe/spectrum-css/assets/63808889/f6c2d389-5337-4cd6-be73-3dc64a987f22)

With Tip, source button to popover tip is 10px but source button to menu border is 16px (pink box)

![Screenshot 2023-11-03 at 9 12 49 AM](https://github.com/adobe/spectrum-css/assets/63808889/b516bd7f-eeec-4a46-8f65-89f39bcf05df)

Reply:
Good catch! I'll look into it, you're probably onto something with the border consideration

Reply:
For the "With Tip" story, I'm measuring 9px between the tip and the button:
![Screenshot 2023-11-08 at 3 33 59 PM](https://github.com/adobe/spectrum-css/assets/965114/f79bfed4-5aba-4187-8a7f-392b6bea1c5b)

Reply:
Ah I see the confusion! Those super light gray 2 pixels outside of what you've selected are part of the tip (the path inside the svg), so if we measure from that it's 8px:

Your image pointing at the pixels I'm talking about:
![measure-these-too](https://github.com/adobe/spectrum-css/assets/23404383/f5f31714-b8f5-493d-8586-cb987aa629d9)

The path selected:
![Screen Shot 2023-11-08 at 4 17 34 PM](https://github.com/adobe/spectrum-css/assets/23404383/e486fae9-6af0-48d2-9580-e328254a2140)

The popover parent element selected:
![Screen Shot 2023-11-08 at 4 19 17 PM](https://github.com/adobe/spectrum-css/assets/23404383/57a81712-adb6-4921-ab72-87e478284302)

Reply:
Hmm, you're sure that was latest @jawinn? I'm seeing 8px on the PR deploy:

![Screen Shot 2023-11-08 at 4 19 17 PM](https://github.com/adobe/spectrum-css/assets/23404383/3eadc0ad-4885-4384-8979-99994b91b60f)